torchgeo/datasets/landcoverai.py

```diff
@@ -236,8 +237,10 @@ def __init__(
            RuntimeError: if ``download=False`` and data is not found, or checksums
                don't match
         """
-        LandCoverAIBase.__init__(self, root, download, checksum)
-        RasterDataset.__init__(self, root, crs, res, transforms=transforms, cache=cache)
+        LandCoverAIBase.__init__(self, cast(str, paths), download, checksum)
```

Comment:
@adamjstewart I added this cast to make mypy happy while this is tested for RasterDatasets only. Do you have a best practice for highlighting this in the code? It will currently crash if LandCoverAIGeo is instantiated with a list, since LandCoverAIBase is not RasterDataset.

Reply:
Is it possible for the base class to also support lists? Like we could just check if list and in that case only use the first value and ignore the rest. Something simple like that.

Reply:
Thanks, I added this temporary fix